kymatio/scattering3d/backend/numpy_backend.py

```diff
@@ -1,31 +1,11 @@
 import numpy as np
 from collections import namedtuple
-from scipy.fftpack import fftn, ifftn
-
+import scipy.fftpack
 
 BACKEND_NAME = 'numpy'
 
-
-def _iscomplex(x):
-    return x.dtype == np.complex64 or x.dtype == np.complex128
-
-
-def complex_modulus(input_array):
-    """Computes complex modulus.
-
-        Parameters
-        ----------
-        input_array : tensor
-            Input tensor whose complex modulus is to be calculated.
-        Returns
-        -------
-        modulus : tensor
-            Tensor the same size as input_array. modulus[..., 0] holds the
-            result of the complex modulus, modulus[..., 1] = 0.
-
-    """
-
-    return np.abs(input_array)
+from ...backend.numpy_backend import modulus, cdgmm, complex_check, real_check
+from ...backend.base_backend import FFT
 
 
 def modulus_rotation(x, module=None):
```

Comment:
not asking for this to be corrected in this PR, but this is an awful function

Reply:
Agreed.

Reply:
The changes you made below don't fix that, right? Looks like it gives the same output, and that's OK for this PR